python/cudf/dataframe/numerical.py

```diff
@@ -382,6 +383,11 @@ def find_last_value(self, value):
             raise ValueError('value not found')
         return found
 
+    def searchsorted(self, value, side='left'):
+        value_col = columnops.as_column(value)
+        out = cpp_search.search_sorted(self, value_col, side)
+        return out.view(NumericalColumn, dtype=out.dtype)
```

Comment:
You shouldn't have to reinterpret this column as a different dtype as the Cython functions will translate the gdf_dtype into the cudf dtype.